# SOME DESCRIPTIVE TITLE.
# Copyright (C) 2019, Qiskit Development Team
# This file is distributed under the same license as the Qiskit package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: Qiskit \n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-06-03 21:59+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: ../../../autodoc/qiskit.validation.exceptions.rst:2
msgid "qiskit.validation.exceptions module"
msgstr ""

#: ../../../../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/qiskit/validation/exceptions.py:docstring of qiskit.validation.exceptions:1
msgid "Exceptions for errors raised by the validation."
msgstr ""

#: ../../../../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/qiskit/validation/exceptions.py:docstring of qiskit.validation.exceptions.ModelValidationError:1
msgid "ベースクラス: :class:`marshmallow.exceptions.ValidationError`, :class:`qiskit.exceptions.QiskitError`"
msgstr ""

#: ../../../../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/qiskit/validation/exceptions.py:docstring of qiskit.validation.exceptions.ModelValidationError:1
msgid "Raised when a sequence subscript is out of range."
msgstr ""

